### DISCUSSION TITLE
could not deploy huginn

### DISCUSSION THREAD

**Discussion by ColtonJakubowski** [domcloud/domcloud#43] [Q&A]
I tried to deploy this app: https://github.com/huginn/huginn, but the runner couldn't create a domain server with an error "xfs_quota: cannot set limits: Function not implemented". Maybe I did something wrong? How can it be fixed?

**Comment by willnode** [comment]
Fixed internally. Thanks for reporting.

Please delete and recreate the website to make it work.
